### DISCUSSION TITLE
How to use client certificates?

### DISCUSSION THREAD

**Discussion by berndlackinger** [ionic-team/capacitor#5005] [Q&A]
I just set a server to require a client certificate, on android Chrome and iOS safari this "just works", but i cannot get it to work with capacitor.
All i found was old cordova plugins. Is there a better way?

**Comment by Hanzofm** [comment]
Any updates about this? I have same requisite

**Comment by metahertz** [comment]
+1 Here, the floccus app (https://github.com/floccusaddon/floccus/commit/f8ad240f62d4c57148ccd5a04c5d568b0df3b7ce uses capacitor/browser) for it's Android app, I believe this is why it does not work with client certificate endpoints. The IOS build and native android browsers load the same server correctly.
